frame.go

```diff
@@ -1469,9 +1470,15 @@ func (f *framer) writeBatchFrame(streamID int, w *writeBatchFrame) error {
 		if w.serialConsistency > 0 {
 			f.writeConsistency(Consistency(w.serialConsistency))
 		}
+
 		if w.defaultTimestamp {
-			now := time.Now().UnixNano() / 1000
-			f.writeLong(now)
+			var ts int64
+			if w.defaultTimestampValue != 0 {
```

Comment:
sometimes it is useful to do an insert with the timestamp set to 0, I guess users could use the `USING TIMESTAMP 0` method, but I think this could be improved, while this problem exists in the query method as well